nion/swift/model/Graphics.py

```diff
@@ -1501,6 +1501,49 @@ def label_position(self, mapping: CoordinateMappingLike, font_metrics: UISetting
         p2 = mapping.map_point_image_norm_to_widget(self.end)
         return Geometry.FloatPoint(y=(p1.y + p2.y) * 0.5, x=(p1.x + p2.x) * 0.5)
 
+    @property
+    def used_role(self) -> typing.Optional[str]:
+        return "fourier_mask"
+    
+    def draw_line(self, r0: int, c0: int, r1: int, c1: int) -> tuple(numpy.array, numpy.array, numpy.array):
```

Comment:
Can you move `draw_line` out of this class (i.e. don't pass `self`)? If the code were left as it were, the return type should be `typing.Tuple[numpy.typing.NDArray[typing.Any], numpy.typing.NDArray[typing.Any], numpy.typing.NDArray[typing.Any]]`. And it would be ideal to define `_ImageDataType = numpy.typing.NDArray[typing.Any]` at the top of the file and make the return type be `typing.Tuple[_ImageDataType, _ImageDataType, _ImageDataType]`.

However, I think this function should be simplified further so that it has the signature:

`def make_line_mask(start: Geometry.FloatPoint, end: Geometry.FloatPoint) -> _ImageDataType: ...`

Notice: it takes a start and end point and return a single mask image.

Reply:
I have taken the method outside the class and simplified it as mentioned in the comment above!